model/export/ResultsExporter.php

```diff
@@ -119,18 +119,21 @@ public function export($destination = null)
     }
 
     /**
+     * @param bool $allColumns
      * @return CallbackTaskInterface
      */
-    public function createExportTask()
+    public function createExportTask(bool $allColumns = false): CallbackTaskInterface
```

Comment:
It is a code smell boolean params in public methods:

I would not use this, otherwise next time that we need a new config, we add a new parameter and suddenly we end up with 10 params and 90% optional.

We can use a command object instead or a simple array

https://martinfowler.com/bliki/FlagArgument.html